File: src/integTest/project/apt/src/main/java/jp/leafytree/android/hello/HelloActivity.java
package jp.leafytree.android.hello;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import org.androidannotations.annotations.AfterViews;
import org.androidannotations.annotations.EActivity;
import org.androidannotations.annotations.ViewById;

@EActivity(R.layout.activity_hello)
public class HelloActivity extends Activity {
    @ViewById
    TextView scalaTextView;

    @AfterViews
    public void fillScalaTextView() {
        scalaTextView.setText(new HelloJava().say() + "\n" + new HelloScala().say());
    }
}

File: src/integTest/app/src/main/scala/jp/leafytree/android/hello/HelloActivity.scala
package jp.leafytree.android.hello

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class HelloActivity extends Activity {
  override def onCreate(savedInstanceState: Bundle) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_hello)
    val scalaTextView = findViewById(R.id.scala_text_view).asInstanceOf[TextView]
    scalaTextView.setText(new HelloJava().say + "\n" + new HelloScala().say)
  }
}
